apollo-portal/src/main/resources/static/i18n/zh-CN.json

```diff
@@ -68,15 +68,24 @@
   "Component.ConfigItem.ModifyItem": "修改配置项",
   "Component.ConfigItem.ItemKey": "Key",
   "Component.ConfigItem.ItemValue": "Value",
-  "Component.ConfigItem.ItemValueTips": "注意: 隐藏字符（空格、换行符、制表符Tab）容易导致配置出错，如果需要检测 Value 中隐藏字符，请点击",
-  "Component.ConfigItem.ItemValueShowDetection": "检测隐藏字符",
-  "Component.ConfigItem.ItemValueNotHiddenChars": "无隐藏字符",
+  "Component.ConfigItem.ItemValueTips": "注意: 特殊字符(空格、换行符、制表符Tab、中文逗号)容易导致配置出错，如果需要检测 Value 中特殊字符，请点击",
```

Comment:
Please also update the [en.json](https://github.com/apolloconfig/apollo/blob/master/apollo-portal/src/main/resources/static/i18n/en.json).

Reply:
> Please also update the [en.json](https://github.com/apolloconfig/apollo/blob/master/apollo-portal/src/main/resources/static/i18n/en.json).

Fixed it.